tox.ini

```diff
@@ -3,8 +3,8 @@ envlist = py39
 skipsdist = True
 
 [flake8]
+; E203 = Whitespace before ':', conflicts with black
 ; W503 = Line break before binary operator
-ignore =
```

Comment:
I assume this was not an intentional deletion?

Reply:
Yup, bad merge.